src/Components/Details/Details.js

```diff
@@ -0,0 +1,58 @@
+import { NavLink, useParams } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import closeArrow from '../imgs/back-arrow.png';
+import './Details.css';
+
+export default function Details() {
+  const { id } = useParams();
+  const cryptos = useSelector((state) => state.cryptos);
+  const PrintCrypto = cryptos.filter((element) => element.id === id);
+  const Crypto = PrintCrypto[0];
+
+  if (PrintCrypto !== undefined) {
+    return (
+      <div className="containerDetails">
+        <NavLink to="/">
+          <img src={closeArrow} alt="Back" />
+        </NavLink>
+        <div
+          className="containerCrypto"
+          id={id}
+        >
+          <img src={Crypto.img} alt={Crypto.name} />
+          <h3>
+            Rank #
+            {Crypto.rank}
```

Comment:
this component along with the other component should have a unit test for the app and some pure functions in Redux code. 💯